### DISCUSSION TITLE
error (Invalid Input c_escape) and error (Invalid Input c_other)

### DISCUSSION THREAD

**Discussion by tridoxx** [GetDKAN/dkan#3874] [Q&A]
hello, I'm getting some datasets, and I get this error, apparently it could be, because the data in the last column is a string separated by spaces that are not inside quotes, so that it looks like a string, but I don't understand what could be the other error, I would be grateful if someone suddenly knows what it refers to, thank you very much
![image](https://user-images.githubusercontent.com/22879539/200995895-da9f5e82-fee4-4417-92b6-da21651d7289.png)

**Comment by janette** [comment]
This error indicates that you may have a parsing issue, confirm that your data file is UTF-8 encoded. Also check the data file for any values that contain a `\`, this is the default escape character and there may be a errant backslash interfering with a quoted value, making the parser fail.

**Reply by tridoxx** [reply]
thanks so much.
